buildscripts/README.md

```diff
@@ -0,0 +1,56 @@
+## Windows
+
+### Prerequisites
+To use the Python build scripts `builddrivers.py` and `buildtools.py`, install Python 3.x. 
```

Comment:
keep it consistent with the other README and also easier to read to have the list in bullets